dartbrcode/lib/dartbrcode.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:ffi' as ffi;
+import 'dart:ffi';
+import 'dart:io';
+import 'package:ffi/ffi.dart';
+
+typedef NativeRustJsonToBrcode = ffi.Pointer<Utf8> Function(ffi.Pointer<Utf8>);
+typedef NativeJsonToBrcode = ffi.Pointer<Utf8> Function();
+final os = Platform.isMacOS ? "libbrcode.dylib" : Platform.isIOS? "libbrcode.a" : "libbrcode.so";
+final ffi.DynamicLibrary dl =
+      ffi.DynamicLibrary.open(os);
+
+String jsonToBrcode(String json) {
```

Comment:
Maybe use `dart:convert` and at least return a `Map<String, dynamic>`, commonly used for JSON objects in dart?

Ideally we could create a `json_serializable` layer on top of it, but that'd would require us to replicate the "schema" in Dart annotations. Maybe we could specify the spec in a configuration file and create a generator for all platforms of this repo?

Reply:
Added this to the todo for a version 1.0.0

Reply:
https://github.com/naomijub/dart-rust-ffi 
Will explore this here